tests/unit/models/cart-model-test.js

```diff
@@ -93,6 +95,25 @@ test('it creates a line item when you add a variant', function (assert) {
   });
 });
 
+test('it returns correct lineItemCount', function (assert) {
+  assert.expect();
```

Comment:
I think `done()` does the work here and is ideal when say the `success` callback has multiple `asserts` while the `error` callback has only one.

But yes, in this scenario, clearly two. You think it should be specified?